### PR TITLE
10 feature push

### PR DESCRIPTION
**Pull request by lukasmatusiewicz**

### CODE REVIEW

File: privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm [hunk 1]
```diff
@@ -83,109 +90,136 @@
                             {
                                 if (otpLength != null && inputObject.value.length === otpLength)
                                 {
-                                    document.getElementById("pi-form-submit-button").click();
```
Comment (nilsbehlen) [inline privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm:86]: please keep the `;`

File: privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm [hunk 2]
```diff
@@ -83,109 +90,136 @@
...
                             <input id="origin" name="origin" value="" type="hidden">
                         </div>
 
-                        #* ALTERNATE TOKENS SECTION *#
+                        #* ALTERNATE TOKENS *#
                         <div class="form-element-wrapper" id="alternateToken" style="padding-top: 20px">
                             <h3 id="alternateTokenHeader">Alternate Login Options</h3>
 
                             <script>
-                                'use strict';
```
Comment (nilsbehlen) [inline privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm:112]: please keep the `;`

File: privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm [hunk 3]
```diff
@@ -83,109 +90,136 @@
...
                                 {
                                     // Submit the form to pass the change to the authenticator
-                                    document.getElementById("mode").value = newMode;
-                                    document.getElementById("pi-form-submit-button").click();
+                                    document.getElementById("mode").value = newMode
+                                    document.getElementById("pi-form-submit-button").click()
+                                }
+                            </script>
+                            <script>
+                                'use strict'
```
Comment (nilsbehlen) [inline privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm:128]: please keep the `;`s

Comment (lukasmatusiewicz) [reply privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm]: fixed

File: privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm (hunk 2 above)
Comment (lukasmatusiewicz) [reply privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm]: fixed

File: privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm (hunk 1 above)
Comment (lukasmatusiewicz) [reply privacyIDEA-impl/src/main/resources/org/privacyidea/views/main.vm]: fixed

**Review by nilsbehlen — APPROVED** [review]

**Pull request merged**